### ISSUE TITLE
Tap position incorrect with iOS in-call status bar

### ISSUE THREAD

**Issue by najeira** [flutter/flutter#12707]
iOS has "in-call status bar" that is extended status bar.
There is a gap between the position of the tap and the widgets.

![47d6ac2022de5fa84b429157685e74b5](https://user-images.githubusercontent.com/1098127/31996060-ec11f9c6-b9c1-11e7-901a-f2b10021c14c.gif)

## Steps to Reproduce

Enable In-call Status bar.
on iOS simulator: Menu > Hardware > Toggle In-call Status bar

## Flutter Doctor

```
[✓] Flutter (on Mac OS X 10.12.6 16G29, locale ja-JP, channel alpha)
    • Flutter at /Applications/flutter
    • Framework revision e8aa40eddd (8 days ago), 2017-10-17 15:42:40 -0700
    • Engine revision 7c4142808c
    • Tools Dart version 1.25.0-dev.11.0

[✓] Android toolchain - develop for Android devices (Android SDK 26.0.1)
    • Android SDK at /Users/najeira/Library/Android/sdk
    • Platform android-26, build-tools 26.0.1
    • ANDROID_HOME = /Users/najeira/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] iOS toolchain - develop for iOS devices (Xcode 9.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.0, Build version 9A235
    • ios-deploy 1.9.2
    • CocoaPods version 1.2.1

[✓] Android Studio (version 2.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] IntelliJ IDEA Ultimate Edition (version 2017.2.5)
    • Flutter plugin version 18.1
    • Dart plugin version 172.4343.25
```

**Comment by cbracken** [comment]
Fixed by @najeira in flutter/engine#4425. Thanks!

**Comment by cbracken** [comment]
Rolled to master in #13523.
